environment.yml

```diff
@@ -0,0 +1,25 @@
+name: nbgrader-dev
```

Comment:
Hmm, I'm tempted to say I'd prefer to try to install everything via pip to make sure that that method of installation still works on windows (I'm more confident that conda will work reliably than pip, TBH). And then we don't have to maintain two separate files with dependencies.